templates/module.njk

```diff
@@ -5,30 +5,37 @@
 {# Create current data dom #}
 {% macro currentData(data) %}
 
-    <div class="current">
-        <div class="large bright light">
-
-            {% set iconURL = module.getIconUrl(config.animateCurrentIcon, data.nww) %}
-            <img class="weather-icon" src="{{ iconURL }}" />
-            <span>{{ data.temp }}°C</span>
-        </div>
-
-        {% if config.showCurrentText %}
-            <div class="medium normal">{{ module.getCurrentText(data.wwtext) | safe }}</div>
-        {% endif %}
-
-        {% if config.showCurrentWindspeed %}
-            <div class="small dimmed">
-                {{ data.ffkmh }} km/h <i class="wi wi-strong-wind"></i>
-
-                {% set windDirKey = module.getWindDirKey(data.dd) %}
-                {% if windDirKey %}
-                    {{ windDirKey | translate }}
-                    <i class="wi wi-wind from-{{data.dd}}-deg fa-fw"></i>
+    {% if config.showCurrentIcon or config.showCurrentTemperature or config.showCurrentText or config.showCurrentWindspeed %}
```

Comment:
I think we can avoid this check because an empty `div` won't use any space.

Reply:
I believe that would depend on whatever nefarious CSS stuff is assigned to the class "current". Personally, I prefer not to generate content that is known to be superfluous, but that's certainly not a fixed rule...